### PR TITLE
Support translation of SearchTerms

### PR DESCRIPTION
**Pull request by tmeschter**
Support translation of `SearchTerms` metadata on properties in CPS rules stored in .xaml format. If we find an attribute named "Value" and it has a sibling attribute named "Name" with a value "SearchTerms", we assume we are looking at the `SearchTerms` metadata on a property.

These search terms are ultimately used by the new property pages UI to support user searches, so we need to be able to localize the terms.

### CODE REVIEW

**Review by drewnoakes — APPROVED** [review]

**Comment by tmeschter** [comment]
@markwilkie xliff-tasks is owned by your team now, correct? Do we need approval from you to merge this in? Also, once this is merged in do you know where new versions of the package will be available?

**Comment by ilyas1974** [comment]
Absolutely, I will have someone look at this today

**Comment by riarenas** [comment]
>Regarding where the package is available, that's a good question

I know this one!

When a PR is merged, this pipeline will publish the packages: https://dnceng.visualstudio.com/internal/_build?definitionId=485

The packages will be published to this feed: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json

The normal usage is that repositories don't consume the xliff-tasks packages directly, and instead do it through Arcade. Arcade gets a dependency update from xliff-tasks every day, which will update the version in use by the Arcade SDK.

**Comment by tmeschter** [comment]
@riarenas Ah, great. Thanks!

Note there are repos that consume xliff-tasks directly, such as dotnet/project-system.

**Comment by riarenas** [comment]
>Note there are repos that consume xliff-tasks directly, such as dotnet/project-system.

Yep in that case it's a matter of making sure to update the version once the official build from this PR publishes the packages to the feed. If your repo is set to restore latest, then it should be automatic, but if it's tied to a specific version, that will need to be updated.

**Comment by tmeschter** [comment]
@ilyas1974 Anything else I need to do before merging this change?

**Comment by ilyas1974** [comment]
I think it is safe to merge. Thank you for checking

**Pull request merged**
